src/views/dialogs/DisplaySettingsDialog.tsx

```diff
@@ -241,21 +237,38 @@ Styled.ColorPreferenceLabel = styled.div`
   gap: 1ch;
 `;
 
-Styled.DiffArrowContainer = styled.div`
+Styled.ArrowIconContainer = styled.div`
   ${layoutMixins.column}
   gap: 0.5ch;
-`;
-
-Styled.DiffArrow = styled(DiffArrow)`
-  --diffArrow-color-positive: var(--color-success);
-  --diffArrow-color-negative: var(--color-error);
 
   svg {
-    width: 0.75em;
     height: 0.75em;
+    width: 0.75em;
   }
 `;
 
+Styled.ArrowIcon = styled(Icon)<{ direction: 'up' | 'down'; color: 'green' | 'red' }>`
+  ${({ direction }) =>
+    ({
+      ['up']: css`
+        transform: rotate(-90deg);
+      `,
+      ['down']: css`
+        transform: rotate(90deg);
+      `,
+    }[direction])}
+
+  ${({ color }) =>
+    ({
+      ['green']: css`
+        color: var(--color-success);
+      `,
+      ['red']: css`
+        color: var(--color-error);
```

Comment:
nit: this works, but I think we can use `--color-green` and `--color-red` since we want the literal non-semantic color

Reply:
ooh yeah - makes sense. I'm going to follow up to add a `color-green` and `color-red` to our `createGlobalStyles` call because there's quite a few places where I actually did this to avoid color inversion 😬  Tracking in TRCL-354